lib/plug/static.ex

```diff
@@ -112,43 +112,43 @@ defmodule Plug.Static do
   end
 
   def init(opts) do
-    at     = Keyword.fetch!(opts, :at)
-    from   = Keyword.fetch!(opts, :from)
-    gzip   = Keyword.get(opts, :gzip, false)
-    brotli = Keyword.get(opts, :brotli, false)
-    only   = Keyword.get(opts, :only, [])
+    at = Keyword.fetch!(opts, :at)
+    from = Keyword.fetch!(opts, :from)
+    gzip? = Keyword.get(opts, :gzip, false)
+    brotli? = Keyword.get(opts, :brotli, false)
+    only = Keyword.get(opts, :only, [])
     prefix = Keyword.get(opts, :only_matching, [])
 
     qs_cache = Keyword.get(opts, :cache_control_for_vsn_requests, "public, max-age=31536000")
     et_cache = Keyword.get(opts, :cache_control_for_etags, "public")
-    headers  = Keyword.get(opts, :headers, %{})
+    headers = Keyword.get(opts, :headers, %{})
 
     from =
-      case from do
-        {_, _} -> from
-        _ when is_atom(from) -> {from, "priv/static"}
-        _ when is_binary(from) -> from
-        _ -> raise ArgumentError, ":from must be an atom, a binary or a tuple"
+      cond do
+        match?({_, _}, from) -> from
+        is_atom(from) -> {from, "priv/static"}
+        is_binary(from) -> from
+        true -> raise ArgumentError, ":from must be an atom, a binary or a tuple"
       end
```

Comment:
Probably doesn't matter in the `init` but the `case` will be faster than the `cond`.